CHANGELOG.md

```diff
@@ -62,7 +62,7 @@ As mentioned above, we not only introduced metapackages in this release, we used
 
 ## Known Issues
 
-No known issues.
+-  Safari (macOS version) does not show toast messages or indicators when Venia switches between online and offline.
```

Comment:
This one really isn't our fault. If we want to drive that point home, `navigator.onLine` always reports `true` in Safari (even when offline), and that's why the toasts & indicators don't appear.

Reply:
Gotcha! Let me add some detail.